cf_units/tests/integration/test_date2num.py

```diff
@@ -24,7 +24,7 @@ def test_single(self):
         res = date2num(date, self.unit, self.calendar)
         # num2date won't return an exact value representing the date,
         # even if one exists
-        self.assertAlmostEqual(exp, res, places=4)
```

Comment:
Now I think about it (rather than just blindly following what `pytestify` gave me) these days `date2num` will give you an `int` if it can - see #202.  I checked and both these tests are indeed getting `numpy.int64` types out.  So seems safe to just use equality.

Reply:
Note: if I had used `pytestify` first, it would have changed it to `assert a == pytest.approx(b)`.

Reply:
😮  Wow.  I wasn't kidding when I said that PyTest was somewhat like normal Python code.
I guess that approx(x) must be a thing with a rather clever comparison behaviour.